docs/reference/sciencedirect/ScDirSubjectClassifications.rst

```diff
@@ -0,0 +1,42 @@
+pybliometrics.scopus.ScDirSubjectClassifications
+===========================================
+
+`ScDirSubjectClassifications()` implements the `Science Direct Subject Classifications API <https://dev.elsevier.com/documentation/SubjectClassificationsAPI.wadl>`_.  It enables retrieval-like queries for All Science Journal Classification (ASJC) subjects/areas.
+
+.. currentmodule:: pybliometrics.sciencedirect
+.. contents:: Table of Contents
+    :local:
+
+Documentation
+-------------
+
+.. autoclass:: ScDirSubjectClassifications
+   :members:
+   :inherited-members:
+
+Examples
```

Comment:
Please add an example how the results can be used to generate a pandas DataFrame. Check ScopusSearch() documentation